db/schema.rb

```diff
@@ -55,8 +55,8 @@
     t.text     "body"
     t.integer  "user_id"
     t.integer  "category_id"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at"
```

Comment:
Nice catch... this was not intentional. I did some housecleaning and reset my database and ran the migrations from master, so maybe it got lost somewhere?